rpc/wrtc_signaling_answerer.go

```diff
@@ -194,19 +194,39 @@ func (ans *webrtcSignalingAnswerer) startAnswerer() {
 			default:
 			}
 			var err error
+			// `newAnswer` opens a bidi grpc stream to the signaling server. But otherwise sends no requests.
 			client, err = newAnswer()
+			var receivedInitRequest = false
 			if err == nil {
-				err = ans.answer(client)
+				// `ans.answer` will send the initial message to the signaling server that says it
+				// is ready to accept connections. Then it waits, typically for a long time, for a
+				// caller to show up. Which is when the signaling server will send a
+				// response. `ans.answer` then follows with gathering ICE candidates and learning of
+				// the caller's ICE candidates to create a working WebRTC PeerConnection. If
+				// successful, the `PeerConnection` + `webrtcServerChannel` will be registered and
+				// available for the `webrtcServer`.
+				receivedInitRequest, err = ans.answer(client)
```

Comment:
I wasn't thinking too hard about options here. I just took the dumbest patch possible that didn't touch the scope/responsibility of this `ans.answer` function call.